### PR TITLE
Fix Oracle Connector

### PR DESCRIPTION
**Pull request by seanzhougoogle**
1. resolve "ORA-01882: timezone region not found" issue
2. add a role property
3. fix the import query in generate spec

### CODE REVIEW

File: oracle-plugin/src/main/java/io/cdap/plugin/oracle/OracleConnectorConfig.java [hunk 1]
```diff
@@ -65,4 +75,18 @@ protected int getDefaultPort() {
...
+    return role == null ? "normal" : role;
+  }
+
+  @Override
+  public Properties getConnectionArgumentsProperties() {
+    Properties prop = super.getConnectionArgumentsProperties();
+    // To solve the "ORA-01882: timezone region not found" issue, see:
+    // https://stackoverflow.com/questions/9156379/ora-01882-timezone-region-not-found
+    prop.put(TIME_ZONE_AS_REGION_PROPERTY, "false");
+    prop.put(INTERNAL_LOGON_PROPERTY, getRole());
```
Comment (yaojiefeng) [inline oracle-plugin/src/main/java/io/cdap/plugin/oracle/OracleConnectorConfig.java:89]: In the stackoverflow question, I don't see this role needs to be included, why do we need to introduce this variable?

File: oracle-plugin/widgets/Oracle-connector.json [hunk 2]
```diff
@@ -64,6 +64,29 @@
...
           "name": "password"
+        },
+        {
+          "label": "Role",
+          "name": "role",
+          "widget-type": "radio-group",
+          "widget-attributes": {
+            "layout": "inline",
+            "default": "normal",
+            "options": [
```
Comment (yaojiefeng) [inline oracle-plugin/widgets/Oracle-connector.json:75]: Are these the only available roles in oracle?

File: oracle-plugin/src/main/java/io/cdap/plugin/oracle/OracleConnectorConfig.java [hunk 3]
```diff
@@ -55,7 +60,12 @@ public String getConnectionString() {
 
   @Name(OracleConstants.CONNECTION_TYPE)
   @Description("Whether to use an SID or Service Name when connecting to the database.")
-  public String connectionType;
+  private String connectionType;
+
+  @Name(OracleConstants.ROLE)
+  @Description("The login role of the user when connecting to the database.")
```
Comment (yaojiefeng) [inline oracle-plugin/src/main/java/io/cdap/plugin/oracle/OracleConnectorConfig.java]: nit - Remove "The"

File: oracle-plugin/src/main/java/io/cdap/plugin/oracle/OracleConnectorConfig.java [hunk 4]
```diff
@@ -55,7 +60,12 @@ public String getConnectionString() {
...
+  @Nullable
+  private String role;
```
Comment (yaojiefeng) [inline oracle-plugin/src/main/java/io/cdap/plugin/oracle/OracleConnectorConfig.java:68]: there is a oracle source md file, can you also add this doc to that file?

File: oracle-plugin/src/main/java/io/cdap/plugin/oracle/OracleConnectorConfig.java (hunk 1 above)
Comment (seanzhougoogle) [reply oracle-plugin/src/main/java/io/cdap/plugin/oracle/OracleConnectorConfig.java:89]: This is irrelevant to the stackoverflow question. 

It is a required property, if you are connecting using a SYSDBA or SYSOPR account, you have to specify this property. Actually for those Oracle sql client, you also need to specify the role

File: oracle-plugin/widgets/Oracle-connector.json (hunk 2 above)
Comment (seanzhougoogle) [reply oracle-plugin/widgets/Oracle-connector.json:75]: Yes.

File: oracle-plugin/src/main/java/io/cdap/plugin/oracle/OracleConnectorConfig.java (hunk 3 above)
Comment (seanzhougoogle) [reply oracle-plugin/src/main/java/io/cdap/plugin/oracle/OracleConnectorConfig.java]: done :)

File: oracle-plugin/src/main/java/io/cdap/plugin/oracle/OracleConnectorConfig.java (hunk 4 above)
Comment (seanzhougoogle) [reply oracle-plugin/src/main/java/io/cdap/plugin/oracle/OracleConnectorConfig.java:68]: done :)

**Review by yaojiefeng — APPROVED** [review]

**Pull request merged**